Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
     name: "Starknet",
     platforms: [
         .iOS(SupportedPlatform.IOSVersion.v13),
-        .macOS(SupportedPlatform.MacOSVersion.v12),
+        .macOS(SupportedPlatform.MacOSVersion.v13),
```

Comment:
Is there a specific reason to bump it? If so, it should be at least mentioned, as we remove support for v12.